cmd/ore/aliyun/aliyun.go

```diff
@@ -44,7 +44,7 @@ func init() {
 		defaultConfigPath = "~/" + auth.AliyunConfigPath
 	}
 
-	Aliyun.PersistentFlags().StringVar(&options.ConfigPath, "config-file", defaultConfigPath, "config file (default \""+defaultConfigPath+"\")")
+	Aliyun.PersistentFlags().StringVar(&options.ConfigPath, "config-file", "", "config file (default \""+defaultConfigPath+"\")")
```

Comment:
If the `config-file` is empty it will automatically be set to the correct path via [this code in `auth/aliyun`](https://github.com/coreos/mantle/blob/master/auth/aliyun.go#L42-L48). This also is currently broken as the `~` isn't correctly expanded into the user directory properly.

Reply:
But the RHCOS pipeline today depends on `ALIYUN_CONFIG_FILE` (related: https://github.com/coreos/mantle/issues/1114).

I think we just need to delete those three lines:

```go
	if defaultConfigPath == "" {
		defaultConfigPath = "~/" + auth.AliyunConfigPath
	}
```

Reply:
Ah, gotcha. Ok pushed.